docs/docusaurus.config.ts

```diff
@@ -43,6 +44,18 @@ const config: Config = {
       {
         excludeAliases: ['console']
       }
+    ],
+    [
+      'client-redirects',
+      {
+        createRedirects(routePath) {
+          if (routePath.includes('/entra')) {
+            return [routePath.replace('/entra', '/aad')];
+          }
+
+          return [];
```

Comment:
It probably comes down to the same thing, but in the examples, they return `undefined` when there should be no redirects. They state you should return a falsy value, `[]` is not a falsy value.

Reply:
The current setup is quite similar to how Docausur implements it itself. So this shouldn't be an issue.